shadowfinder/shadowfinder.py

```diff
@@ -12,27 +12,47 @@
 
 class ShadowFinder:
     def __init__(
-        self, object_height=None, shadow_length=None, date_time=None, time_format="utc"
+        self,
+        object_height=None,
+        shadow_length=None,
+        date_time=None,
+        time_format="utc",
+        sun_altitude_angle=None,
     ):
 
-        self.set_details(object_height, shadow_length, date_time, time_format)
+        print(f"object_height: {object_height}")
+        print(f"shadow_length: {shadow_length}")
+        print(f"date_time: {date_time}")
+        print(f"time_format: {time_format}")
+        print(f"sun_altitude_angle: {sun_altitude_angle}")
+
+        self.set_details(
+            object_height, shadow_length, date_time, time_format, sun_altitude_angle
+        )
 
         self.lats = None
         self.lons = None
-        self.shadow_lengths = None
+        self.correct_location_probabilities = None
```

Comment:
I like the refactor of this, would you mind switching it to `location_likelihoods` a bit shorter and hopefully disambiguates between "correct location probabilities" and "correct-location probabilities"

Reply:
naming, "a programmers worst nightmare" xD